spark/v2.4/spark/src/test/java/org/apache/iceberg/spark/data/TestSparkParquetReadMetadataColumns.java

```diff
@@ -70,8 +70,7 @@ public class TestSparkParquetReadMetadataColumns {
   private static final Schema PROJECTION_SCHEMA = new Schema(
       required(100, "id", Types.LongType.get()),
       required(101, "data", Types.StringType.get()),
-      MetadataColumns.ROW_POSITION,
-      MetadataColumns.IS_DELETED
```

Comment:
Why we change these unit tests?

Reply:
Some tests(e.g., `testReadRowNumbersWithDelete`) failed due to it uses the `PROJECTION_SCHEMA`, which contains IS_DELETED. So the row count was wrong since it still assume only non-deleted rows are there.  We can either remove the column `IS_DELETED` so that deleted rows won't be in the results. Or we can change the test case to filter out the deleted rows in the results. I choose the former so that it can fix multiple failure at once.